bot_cli/address_book_classes.py

```diff
@@ -146,10 +146,9 @@ def custom_serializer(obj):
 
             json.dump(self.data, file, default=custom_serializer, indent=4)
 
-    def get_birthdays_per_week(self, period: int = 7) -> dict | None:
-        users_dictionary_per_weekday = defaultdict(list)
+    def get_birthdays_per_period(self, period: int = 7) -> dict | None:
+        users_to_celebrate = defaultdict(list)
```

Comment:
`upcoming_birthdays` might be a better name

Reply:
done